doc/classes/VisualShaderNodeColorConstant.xml

```diff
@@ -1,15 +1,19 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="VisualShaderNodeColorConstant" inherits="VisualShaderNode" category="Core" version="3.2">
 	<brief_description>
+		A [Color] constant to be used within the visual shader graph.
 	</brief_description>
 	<description>
+		Has two output ports representing rgb and alpha channels of color.
+		Translated to "vec3 rgb" and "float alpha" in the shader language.
```

Comment:
"RGB and alpha channels of [Color]."
"[code]vec3 rgb[/code] and [code]float alpha[/code]"